Base/QTCLI/vtkSlicerCLIModuleLogic.h

```diff
@@ -76,6 +76,9 @@ class Q_SLICER_BASE_QTCLI_EXPORT vtkSlicerCLIModuleLogic :
   void SetDeleteTemporaryFiles(int value);
   int GetDeleteTemporaryFiles() const;
 
+  void SetAllowMemoryTransfer(int value);
```

Comment:
I think `AllowInMemoryTransfer` would be a bit more clear name. Otherwise I like having this option (and would even consider removing in-memory transfer at some point).